lib/deck.rb

```diff
@@ -0,0 +1,20 @@
+class Deck
+
+    attr_reader :cards
+
+def initialize(cards) 
+    @cards  = cards
+end
+
+
+def count
+    cards.count
+end
+
+def cards_in_category(category)
+@cards.find_all do |card|
+card.category == category
+end
+end
+
+end
```

Comment:
Careful with your indentation. make sure def's, and tabbed over, and also the content inside of them: 

class Deck 

       def initialize()
               code code code 
       end 

end